sprockets/logging.py

```diff
@@ -18,11 +18,12 @@
 import traceback
 
 try:
-    from tornado import log
-except ImportError:
+    from tornado import escape, log
+except ImportError:  # pragma no cover
+    escape = None
```

Comment:
When importing [sprockets.logging from setup.py](https://github.com/sprockets/sprockets.logging/blob/fix-query-arg-handling/setup.py#L7) without having tornado installed.  This makes it possible to import the module to retrieve the version number.  If you try to do anything else without tornado installed, then it will fail spectacularly.

This is safe because you cannot actually install it without installing tornado ... or at least you shouldn't be able to.  Lemme adjust *setup.py* to actually require it :persevere: